packages/core/content-manager/admin/src/history/components/VersionInputRenderer.tsx

```diff
@@ -345,6 +347,17 @@ const VersionInputRenderer = ({
     edit: { components: componentsLayout },
   } = useDocLayout();
 
+  const rulesEngine = createRulesEngine();
+  const attribute = props.attribute;
+
+  if (attribute?.conditions && attribute?.conditions.visible) {
+    const isVisible = rulesEngine.evaluate(attribute?.conditions.visible, fieldValues);
```

Comment:
A little confusing to read with `isVisible` here and `visible` below. Maybe some better variable names would make it clearer that the field is hidden because of a condition?